projects/packages/jetpack-mu-wpcom/changelog/launchpad-fullscreen-method-refactor

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: changed
+Comment: simple refactor
```

Comment:
I think it's worth being explicit about the refactor.
```suggestion
Comment: Rename 'wpcom_get_launchpad_is_enabled()' to 'wpcom_launchpad_get_fullscreen_enabled()'
```

Reply:
Completely agree 😁